web/packages/teleport/src/Roles/RoleEditor/RoleEditorAdapter.tsx

```diff
@@ -70,7 +70,7 @@ export function RoleEditorAdapter({
   }, [originalContent]);
 
   const onRoleUpdate = useCallback(
-    debounce(roleDiffProps.updateRoleDiff, 500),
+    debounce(role => roleDiffProps?.updateRoleDiff(role), 500),
```

Comment:
i tried to just do the conditional inline but `debounce` required the method to exist rather than optionally receiving an undefined function